pkg/api/v1/schema_swagger_generated.go

```diff
@@ -13,6 +13,7 @@ func (CloudInitNoCloudSource) SwaggerDoc() map[string]string {
 
 func (DomainSpec) SwaggerDoc() map[string]string {
 	return map[string]string{
+		"":          "+k8s:openapi-gen=true",
```

Comment:
These artifacts do not look right.

Reply:
hack/generate.sh takes care of this after the fact using sed.

All our 'generators' have been making it into the swagger descriptions. This isn't anything new that this patch introduced. At some point I think the swagger generator should filter out lines that start with '+k8s:'.